crates/bevy_ui/src/widget/text.rs

```diff
@@ -48,18 +57,6 @@ impl Default for TextNodeFlags {
     }
 }
 
-/// [`TextBundle`] was removed in favor of required components.
```

Comment:
This definitely shouldn't be removed in this PR.

Reply:
I figured this would be in 0.16 anyway, but I can make that a separate PR.

Reply:
Yeah, separate PR please!